CHANGELOG.md

```diff
@@ -4,5 +4,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.2] - 2017-03-14
+Corrected API docs.
+
+### Fixed
+- Issue where CodeClimate would consider underscore-prefixed private methods to
+be breaking the naming convention.
```

Comment:
Was this actually a fix for _this_ package, given that there are no underscore-prefixed structure elements in this package?